packages/eslint-plugin-pf-codemods/lib/rules/v5/contextSelector-update-deprecatedPath.js

```diff
@@ -2,8 +2,16 @@ const { moveSpecifiers } = require("../../helpers");
 
 const specifiersToMove = [
   "ContextSelector",
+  "ContextSelectorProps",
+  "ContextSelectorContext",
   "ContextSelectorItem",
+  "ContextSelectorItemProps",
   "ContextSelectorFooter",
+  "ContextSelectorFooterProps",
+  "ContextSelectorMenuList",
+  "ContextSelectorMenuListProps",
+  "ContextSelectorToggle",
+  "ContextSelectorToggleProps"
```

Comment:
We should be able to omit some of these since they're not being exported in the index.js file

```suggestion
  "ContextSelector",
  "ContextSelectorProps",
  "ContextSelectorItem",
  "ContextSelectorItemProps",
  "ContextSelectorFooter",
  "ContextSelectorFooterProps",
```